WikiPreview.Fluent.Plugin/WikiPreviewSearchApp.cs

```diff
@@ -63,17 +65,19 @@ public ValueTask<IHandleResult> HandleSearchResult(ISearchResult searchResult)
                 IProcessManager managerInstance = ProcessUtils.GetManagerInstance();
                 string actionUrl = wikiPreviewSearchOperation.ActionType switch
                 {
-                    ActionType.Wikipedia => WikiRootUrl + displayedName,
-                    ActionType.Wikiwand => WikiWandUrl + displayedName,
-                    ActionType.GoogleSearch => GoogleSearchUrl + displayedName,
+                    ActionType.Wikipedia => new StringBuilder(WikiRootUrl).Append(wikiPreviewSearchResult.Url)
```

Comment:
Don't use `StringBuilder` to append a single string, the memory allocation won't be worth it in terms of performance.
Also, the code is less readable this way.